Alive/Controllers/Alive.php

```diff
@@ -146,30 +146,27 @@ private function assertCacheStatus() :void {
 	private function assertBridgeStatus() :void {
 
 		// if test requested in the configuration file
-		if( Config::get('alive', 'assert_bridge_status') !== null && Config::get('alive', 'assert_bridge_status') == 1 ) {
-			$Request_query = new Query;
-			$Request_query->query( 
+		if( Config::get('alive', 'assert_bridge_status') == 1 ) {
+			$bridge_query = (new Query)
+			->query( 
 				'SELECT "_id_site", "'. Config::get('alive','bridge_expected_column') .'" '.
 				'FROM "'. Config::get('alive','bridge_table') .'" WHERE "_id_site" = '.
 				Config::get('alive','bridge_id') 
 			);
-			$response = Bridge::execute( $Request_query );
+			$response = Bridge::execute( $bridge_query );
 			// if test failed as a crash
 			if( is_null($response) ) {
 				Throw new Exception("Can't access the Filemaker database");
 			}
 			// if an error is found in the request, return it
 			elseif( isset($response['error'] ) ) {
 				Throw new Exception( $response['error'] );
-
 			} 
-			else {
-				if( 
-					$response[0]->get( Config::get('alive','bridge_expected_column_value') ) != 
-					Config::get('alive',' bridge_expected_column_value') 
-				) {
-					Throw new Exception("The object of the Filemaker database has a wrong match value");
-				}
+			elseif( 
+				$response[0]->get( Config::get('alive','bridge_expected_column_value') ) != 
+				Config::get('alive',' bridge_expected_column_value') 
+			) {
+				Throw new Exception("The object of the Filemaker database has a wrong match value");
```

Comment:
Pas très grammaticalement valide ça ;)
`The object's column value differs from what we were expecting`